sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -1012,13 +1008,24 @@ export class ServiceBusMessageImpl implements ReceivedMessageWithLock {
     if (error) {
       log.error(
         "[%s] An error occurred when renewing the lock on the message with id '%s': %O",
-        this._context.namespace.connectionId,
+        this._context.connectionId,
         this.messageId,
         error
       );
       throw error;
     }
-    this.lockedUntilUtc = await this._context.managementClient!.renewLock(this.lockToken!);
+
+    if (this.delivery.link) {
+      const associatedReceiver = ConnectionContext.getReceiverFromCache(
```

Comment:
Why isn't .getReceiverFromCache just a member function of this._context?

Reply:
There was a reason for this which I cannot recall anymore... Updated as per suggestion in https://github.com/Azure/azure-sdk-for-js/pull/9710/commits/612869e79210efbe1b13c3ad733afb951ec64ddc